test/liblll/Parser.cpp

```diff
@@ -174,6 +174,16 @@ BOOST_AUTO_TEST_CASE(list)
 	BOOST_CHECK(!successParse("()"));
 }
 
+BOOST_AUTO_TEST_CASE(returndatacopy)
+{
+	BOOST_CHECK(successParse("( returndatacopy 0 1 2 )"));
```

Comment:
I agree.  The test succeeded even before the change. So this does not add any value.